tutorials/quickstart.eve.md

```diff
@@ -0,0 +1,90 @@
+# Quickstart
+
+In this document we're going to give a 5 minute introduction to concepts essential to Eve. If you've never used Eve before, you're in the right place.
+
+## Adding records to Eve
+
+Eve represents data as records, key value pairs attached to a unique ID. Eve programs are made up of small composable blocks that search for and create records. Let's start with a block that adds a record to Eve:
+
+```eve
+commit
+  [#greeting text: "hello world"]
+```
+
+This record is tagged `#greeting`, and has an attribute "text" with the value "hello world". Although tags are attributes like any other,they have special syntax because they are used often.
+
+## Finding records in Eve
+
+We can search for records, and then bind new records to those that are found. Eve finds every record matching the supplied patterns, and then binds records dependent on them. If no records match the search, then the block does not run at all. A block will only run if every record in a search matches at least one result. Let's search for the `#greeting` we just committed, and then display it in a div:
+
+```eve
+search
+  [#greeting text]
+
+bind
+  [#ui/div text]
+```
+
+Names are identities in Eve; because they have the same name, the `text` in `#greeting` is bound to the `text` in the div. Go ahead and add another `#greeting` record to the first block to see what happens when more than one record matches the search. 
+
+## Records update as data changes
+
+Blocks in Eve react automatically to changes in data. When a record changes, any bound records are automatically updated. Let's search for the current time, and display it in a div:
+
+```eve
+search
+  [#time seconds]
+
+bind 
+  [#ui/div text: seconds]
+```
+
+As the time changes, the output updates to reflect the current state of the `#time` record. Records can also be committed in response to a `search`, but the behavior is a little different. Try changing `bind` to `commit` in the above block and see what happens.
```

Comment:
I would leave off "in response to a search" since that catches my attention more than the word committed, seeming like there's some special way to do an action "in response to a search" which differs from what you showed above.